packages/core/src/storage/layout.ts

```diff
@@ -9,13 +9,17 @@ export interface StorageLayout {
   layoutVersion?: string;
   storage: StorageItem[];
   types: Record<string, TypeItem>;
+  flat?: boolean;
 }
 
 export interface StorageItem<Type = string> {
+  astId?: number;
```

Comment:
We don't want to store ast ids. I don't think we need them.

Reply:
Oh from when we read these objects from solc's `storageLayout`. I see.

We need this optional field here but we don't want